### ISSUE TITLE
cluster family or clusters-family?

### ISSUE THREAD

**Issue by icy** [oracle/cluster-api-provider-oci#273]
https://github.com/oracle/cluster-api-provider-oci/blob/18012bfa12cb7f4bcb50366e53d7ac9061350eed/docs/src/gs/iam/iam-oke.md?plain=1#LL10C42-L10C56

I don't think it's cluster family. That took me a lot of time to figure out. Pls confirm. Thx

**Comment by shyamradhakrishnan** [comment]
@icy sorry for the bug, it is cluster-family. We will fix this.

**Comment by shyamradhakrishnan** [comment]
please go through OKE docs https://docs.oracle.com/en-us/iaas/Content/ContEng/Concepts/contengpolicyconfig.htm for any fine grained permissions, we will link the doc in the book as well.

**Comment by icy** [comment]
> please go through OKE docs https://docs.oracle.com/en-us/iaas/Content/ContEng/Concepts/contengpolicyconfig.htm for any fine grained permissions, we will link the doc in the book as well.

Thank you for the feedback.

Normally I use that page (docs.oracle.com) but somehow today I got the link to the github page, and I think it's better. At least, I can report a bug here . 

I reported via some private channel a problem with our official blog: but no one really cares? this page is totally broken on all browers I have (win/linux): https://blogs.oracle.com/mysql/post/point-in-time-recovery-in-oci-mds-with-object-storage-part-2
